src/auth/token-generator.ts

```diff
@@ -297,8 +299,13 @@ export class FirebaseTokenGenerator {
       }
     }
     return this.signer.getAccountId().then((account) => {
+      // When communicating with the Auth Emulator we don't sign custom tokens which means:
```

Comment:
What about the call to `getAccountId()` which may also make rpc calls?

Reply:
Yeah moved this into an implementation.